crates/rpc-client/src/client.rs

```diff
@@ -118,15 +148,44 @@ pub struct RpcClientInner<T> {
 
 impl<T> RpcClientInner<T> {
     /// Create a new [`RpcClient`] with the given transport.
+    #[inline]
     pub const fn new(t: T, is_local: bool) -> Self {
         Self { transport: t, is_local, id: AtomicU64::new(0) }
     }
 
+    /// Returns the default poll interval for the client.
+    pub const fn default_poll_interval(&self) -> Duration {
+        if self.is_local {
```

Comment:
should make this configurable eventually, follow up material

Reply:
https://github.com/alloy-rs/alloy/issues/290